aviary/examples/external_subsystems/battery/battery_builder.py

```diff
@@ -250,7 +250,7 @@ def preprocess_inputs(self, aviary_inputs):
         In this case, it sets the values battery performance based on the battery cell type.
         '''
         battery_cell_info_18650 = {
-            Aircraft.Battery.Cell.DISCHARGE_RATE: [10.5, 'A'],
+            Aircraft.Battery.Cell.DISCHARGE_RATE: [2.0, 'A'],
```

Comment:
What are these value changes in the battery builder? 10.5 -> 2 is quite a difference.